src/devices/SocketCan/CanRaw.cs

```diff
@@ -55,11 +56,10 @@ public void WriteFrame(ReadOnlySpan<byte> data, CanId id)
             {
                 Span<byte> frameData = new Span<byte>(frame.Data, data.Length);
                 data.CopyTo(frameData);
-            }
 
-            ReadOnlySpan<CanFrame> frameSpan = MemoryMarshal.CreateReadOnlySpan(ref frame, 1);
-            ReadOnlySpan<byte> buff = MemoryMarshal.AsBytes(frameSpan);
-            Interop.Write(_handle, buff);
+                byte* buff = (byte*)&frame;
```

Comment:
Where you able to test this? Is the problem here that Netstandard2.0 doesn't have these MemoryMarshal? Aren't these methods available through the System.Memory package?

Reply:
No, I could not test this, but I'm quite certain that it's equivalent. I did not look for an implementation of these methods because the changed code is simpler than the old. This converted to two different ReadOnlySpan types and finally to a byte*, while you can just directly take the address of the frame.

Reply:
Ok, I checked and they don't seem to be available on System.Memory package for NS2.0. To be honest, it feels a bit weird to change this when we haven't tested it at all even when we have high confidence on the change, but for now let's keep it and if folks using the binding find an issue here they can log the issue and we can investigate then.

Reply:
The binding was originally provided by @krwq I think. So maybe he has the hardware available to test it?